### ISSUE TITLE
hyprpick and swaylock doesnt work on monitors with bitdepth 10

### ISSUE THREAD

**Issue by TheyCallMeSecond** [prasanthrangan/hyprdots#1834]
looks like again hyprpick and swaylock have problem with monitors with bitdepth 10
probably its because of wayland update (or xdg-desktop-portal-hyprland)

**Comment by rishav12s** [comment]
yes screenshot using grim also doesnt work on bitdepth 10 upstream bug ig

**Comment by kRHYME7** [comment]
try using hyprland-git to check if issue is fixed
